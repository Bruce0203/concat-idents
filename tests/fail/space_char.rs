concat_idents::concat_idents!(fn_name = ' ', name {
    fn fn_name() {
        /* idents cannot contain spaces */
    }
});

fn main() {}
