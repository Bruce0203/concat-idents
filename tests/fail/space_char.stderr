error: character literals can only contain [a-zA-Z0-9_]
 --> $DIR/space_char.rs:1:41
  |
1 | concat_idents::concat_idents!(fn_name = ' ', name {
  |                                         ^^^
